clap_complete/src/dynamic/shells/mod.rs

```diff
@@ -27,6 +107,70 @@ pub enum CompleteCommand {
 }
 
 /// Generally used via [`CompleteCommand`]
+///
+/// This is the arguments for the `complete` subcommand.
+///
+/// **NOTE**: We should replace the `completion.bash` to filename we wanted, also `completion.fish`, `completion.zsh`, `completion.ps1`, `completion.elv`.
+///
+/// To generate shell completion scripts, we can use the `complete` subcommand as follows:
```

Comment:
This focuses on
- Step
  - Shell

We likely should instead do
- Shell
  - Step

As someone using a a shell will not care about details from other shells

Reply:
Thanks for your review! My way does make it harder for the user to read.